cmd/etherfi/registerBLS.go

```diff
@@ -19,10 +19,11 @@ import (
 )
 
 var (
-	registryCoordinatorMainnet = common.HexToAddress("0x0BAAc79acD45A023E19345c352d8a7a83C4e5656")
-	registryCoordinatorHolesky = common.HexToAddress("0x53012C69A189cfA2D9d29eb6F19B32e0A2EA3490")
-	operatorManagerMainnet     = common.HexToAddress("0x2093Bbb221f1d8C7c932c32ee28Be6dEe4a37A6a")
-	operatorManagerHolesky     = common.HexToAddress("0xdf9679e8bfce22ae503fd2726cb1218a18cd8bf4")
+	// TODO: I think the registry coordinator needs to be a provided paramater?
```

Comment:
agree. currently this command will work for EigenDA only.

Reply:
this is all fixed and cleaned up in a commit not on this tree. I'll get everything synced up in the morning